scripts/dev

```diff
@@ -323,8 +323,8 @@ namespace :db do
     sh "docker-compose start db_migrate"
     loop do
       sleep 0.5
-      status = `docker-compose ps | grep "db_migrate_1"`
-      match = /Exit (\d+)/.match(status)
+      status = `docker-compose ps | grep "db_migrate-1"`
+      match = /exited \((\d+)\)/.match(status)
```

Comment:
Was this what was causing the backend to sometimes never spin up, as it never finished waiting for the migraitons?

Reply:
I wish it was! I don't think it will help that unfortunately, this is specifically with the `db:migrate` command. I don't think the up command calls this script. I ended up recreating the db and migrating it quite a few times, and this command was not analyzing if the migration was complete correctly. (Thanks to Riley for help with the Ruby REGEX!)